drivers/ethernet/Kconfig.renesas_ra

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config ETH_RENESAS_RA
+	bool "Renesas RA Ethernet"
+	default y
+	select USE_RA_FSP_ETHER
```

Comment:
if your eth driver needs the mdio phy consider adding `select MDIO` or `imply MDIO`, so users don't have to activate it manual.

Reply:
@maass-hamburg It's added